src/test/py/test_crash_recovery.py

```diff
@@ -218,6 +218,8 @@ def test_activate_deactivate(pipeline, clean_db):
   cur.close()
   conn.close()
 
+  time.sleep(2)
```

Comment:
Yes, the process deactivation now takes longer. I suspect there was a race beforehand that is now being exposed.